### DISCUSSION TITLE
What is environment value WG_CONFIG_PORT for?

### DISCUSSION THREAD

**Discussion by instantdreams** [wg-easy/wg-easy#1100] [General]
It is not documented - what does it do?

**Comment by pheiduck** [comment]
It's used for a Homeassistent Add-on see:
https://github.com/adriy-be/homeassistant-addons-jdeath/commit/15271c85c13d9b9fd9adfbc250943266cb608da2

It's not required for normal use cases.

**Comment by instantdreams** [comment]
Thank you. Hopefully that'll get updated in the documentation soon.
